front/src/modules/ui/table/components/EntityTableHeaderOptions.tsx

```diff
@@ -0,0 +1,96 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import { DropdownButton } from '@/ui/dropdown/components/DropdownButton';
+import { StyledDropdownMenu } from '@/ui/dropdown/components/StyledDropdownMenu';
+import { StyledDropdownMenuItemsContainer } from '@/ui/dropdown/components/StyledDropdownMenuItemsContainer';
+import { useDropdownButton } from '@/ui/dropdown/hooks/useDropdownButton';
+import { ViewFieldMetadata } from '@/ui/editable-field/types/ViewField';
+import {
+  IconArrowNarrowLeft,
+  IconArrowNarrowRight,
+  IconEyeOff,
+} from '@/ui/icon';
+import { MenuItem } from '@/ui/menu-item/components/MenuItem';
+
+import { ColumnHeaderDropdownId } from '../constants/ColumnHeaderDropdownId';
+import { useTableColumns } from '../hooks/useTableColumns';
+import { ColumnDefinition } from '../types/ColumnDefinition';
+
+const StyledDropdownContainer = styled.div`
+  left: 0px;
+  position: absolute;
+  top: 32px;
+  z-index: 1;
+`;
+type OwnProps = {
```

Comment:
our new convention is to name it after the component name: ColumnHeadOptionDropdownProps here